products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,381 @@
+and `year` (e.g. `2019`).
+
+Till this point we only described nodes. Now we need to describe how those nodes are connected.
+
+* `: TRANSFERRED_FROM` - connects team node `Team` to node `Transfer` representing a team
+where the player is being transferred from.
+* `: TRANSFERRED_TO` - connects node `Transfer` to team node `Team` where player is being transferred to.
+* `: TRANSFERRED_IN` - connects player node `Player` to node `Transfer` representing a player
+that was transferred in the connected transfer.
+* `: HAPPENED_IN` - connects node `Transfer` to the node `Season` in which transfer has happened.
+* `: PLAYS_IN` - connects node `Team` that plays in league node `League`.
+
+
+![](../data/FootballTransfers_metagraph.png)
+
+### Importing the Snapshot
+
+We have prepared a database snapshot for this example, so you can easily
+import it when starting Memgraph using the `--data-directory` option.
+
+```plaintext
+sudo -u memgraph \
+  /usr/lib/memgraph/memgraph --data-directory /usr/share/memgraph/examples/TEDTalk \
+    --storage-snapshot-interval-sec=0 --storage-wal-enabled=false \
+    --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+```
+
+When using Memgraph installed from a Debian or RPM package, 
+the currently running Memgraph server needs to be stopped before 
+importing the example, using the following command:
+
+```plaintext
+systemctl stop memgraph
+```
+
+When using Docker, the example can simply be imported with the following command:
+
+```plaintext
+docker run -p 7687:7687 \
+  -v mg_lib:/var/lib/memgraph -v mg_log:/var/log/memgraph -v mg_etc:/etc/memgraph \
+  memgraph --data-directory /usr/share/memgraph/examples/TEDTalk \
+  --storage-snapshot-interval-sec=0 --storage-wal-enabled=false \
+  --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+```
+
+You should note that any modifications of the database state will persist
+only during this run of Memgraph.
+
+### Example Queries using OpenCypher
+
+In the queries below, we are using [OpenCypher](https://www.opencypher.org) 
+to query Memgraph via the console.
+
+Now when we have a dataset of football transfers from season 1992/1993 to season 2019/2020
+loaded in Memgraph, we are ready to gain some information out of it.
+
+
+1) Let's say you want to find 20 most expensive transfers.
+As mentioned before, transfers fees are represented in millions of euros.
+
+```opencypher
+MATCH 
+    (t:Transfer)<-[:TRANSFERRED_IN]-(p:Player)
+WHERE 
+    t.fee is NOT NULL
+RETURN ROUND(t.fee) + 'M €' as transfer_fee, p.name AS player_name
+ORDER BY t.fee DESC LIMIT 20
+```
+
+2) What about finding the most expensive transfer per season? 
+
+```opencypher
+MATCH 
+    (s:Season)<-[:HAPPENED_IN]-(t:Transfer)<-[:TRANSFERRED_IN]-(:Player)
+WHERE 
+    t.fee is NOT NULL
+WITH  s.name as season_name, MAX(t.fee) as max_fee
+RETURN  ROUND(max_fee) + 'M €' as max_transfer_fee, season_name
```

Comment:
```suggestion
WITH s.name as season_name, MAX(t.fee) as max_fee
RETURN ROUND(max_fee) + 'M €' as max_transfer_fee, season_name
```
There are still issues here. You can click on Apply or commit suggestion and it will be applied with all the suggested changes. Go through the latest PR comments with suggestions and check if everything is added correctly.